### PR TITLE
Fixes team stats

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes team stats

fixes https://github.com/EveripediaNetwork/issues/issues/932

### CODE REVIEW

File: src/components/Oracles/MarchMadness/TeamStatsViewer.tsx
Comment (Emmanueldmlr) [inline src/components/Oracles/MarchMadness/TeamStatsViewer.tsx:32]: Can you move this to the data folder.

Comment (Adebesin-Cell) [reply src/components/Oracles/MarchMadness/TeamStatsViewer.tsx:32]: Okay, I’ll work on it now.

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
- [x] There is still this weird thing below all the subpages on march madness
image
- [x] Horizontal scrolling issue is fixed now, but on larger screens where the whole page is visible, some parts of the page are cut off
image
- [x] There are 3 columns on the table, but only 2 is visible on the team statistics page
image
- [x] on the press page, the cards with no pictures were not implemented well. check figma to see how the cards are designed
image
- [x] The links on the press page are meant to be clickable too, and the arrangement of the press page is not yet implemented

**Comment by Adebesin-Cell** [comment]
@Simplypheyie96 About the break on the larger screens is to prevent layout issues, so you’d have to scroll horizontally. I’ll look into it if it won’t break the layout by rendering the entire tournament on large screens.

**Comment by Adebesin-Cell** [comment]
@Simplypheyie96 about the weird colors, not sure why it’s happening. I’ll inspect closely cause I can’t seem to replicate it here.

**Comment by Simplypheyie96** [comment]
> Visit Preview

i see. you might want to do something about that.

**Comment by Simplypheyie96** [comment]
> @Simplypheyie96 about the weird colors, not sure why it’s happening. I’ll inspect closely cause I can’t seem to replicate it here.

on the brackets page, it shows as a container for the Horizontal scrolling, but it still not looking good even as a scroll container
image

**Review by Royal-lobster — COMMENTED** [review]
@Adebesin-Cell there seems to be a long delay for first time users to render the stats panel, i think a loading state here would cover the wait time. 

CleanShot 2022-12-28 at 18 07 40@2x

**Comment by Adebesin-Cell** [comment]
> @Adebesin-Cell there seems to be a long delay for first time users to render the stats panel, i think a loading state here would cover the wait time.
> 
> CleanShot 2022-12-28 at 18 07 40@2x

Ha! okay. I'll work on it.

**Comment by Adebesin-Cell** [comment]
> > @Simplypheyie96 about the weird colors, not sure why it’s happening. I’ll inspect closely cause I can’t seem to replicate it here.
> 
> on the brackets page, it shows as a container for the Horizontal scrolling, but it still not looking good even as a scroll container image

I've tried checking this but I didn't get any replicate.

**Comment by Simplypheyie96** [comment]
> > > @Simplypheyie96 about the weird colors, not sure why it’s happening. I’ll inspect closely cause I can’t seem to replicate it here.
> > 
> > 
> > on the brackets page, it shows as a container for the Horizontal scrolling, but it still not looking good even as a scroll container image
> 
> I've tried checking this but I didn't get any replication. @Simplypheyie96

what do you think the issue is? Maybe it will not be there after merge

**Review by Simplypheyie96 — APPROVED** [review]
Everything is fixed now, except the sizes for the press page and the arrangement.

**Comment by Adebesin-Cell** [comment]
> > > > @Simplypheyie96 about the weird colors, not sure why it’s happening. I’ll inspect closely cause I can’t seem to replicate it here.
> > > 
> > > 
> > > on the brackets page, it shows as a container for the Horizontal scrolling, but it still not looking good even as a scroll container image
> > 
> > 
> > I've tried checking this but I didn't get any replication. @Simplypheyie96
> 
> what do you think the issue is? Maybe it will not be there after merge

Can't really tell. After merge, maybe it'll fix.

**Comment by Simplypheyie96** [comment]
> > > > > @Simplypheyie96 about the weird colors, not sure why it’s happening. I’ll inspect closely cause I can’t seem to replicate it here.
> > > > 
> > > > 
> > > > on the brackets page, it shows as a container for the Horizontal scrolling, but it still not looking good even as a scroll container image
> > > 
> > > 
> > > I've tried checking this but I didn't get any replication. @Simplypheyie96
> > 
> > 
> > what do you think the issue is? Maybe it will not be there after merge
> 
> Can't really tell. After merge, maybe it'll fix.

it is no longer there now that I checked.

**Comment by Adebesin-Cell** [comment]
> Everything is fixed now, except the sizes for the press page and the arrangement.

Yeah, I'm working on the arrangement already.

**Comment by Adebesin-Cell** [comment]
@Simplypheyie96 You can inspect the press view card arrangement now.

File: src/components/Oracles/MarchMadness/TeamStatsViewer.tsx
Comment (Royal-lobster) [inline src/components/Oracles/MarchMadness/TeamStatsViewer.tsx:36]: The dependency array needs average and total so tableRows gets updated again once the component renders with new average or total info, the warning should be fixed after including them.

**Review by Royal-lobster — CHANGES_REQUESTED** [review]

File: src/components/Oracles/MarchMadness/TeamStatsViewer.tsx
Comment (Adebesin-Cell) [reply src/components/Oracles/MarchMadness/TeamStatsViewer.tsx:36]: Okay, I'll push the fix.

**Comment by Adebesin-Cell** [comment]
Fixed the warning. I think we are good to go after @Simplypheyie96 reviews the press card arrangements.

**Review by Royal-lobster — APPROVED** [review]

**Review by Emmanueldmlr — APPROVED** [review]

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell , loading this page https://oraqles-908uo3d0r-prediqt.vercel.app/oracles/march-madness for the first time takes a lot of time, maybe you should add a loader to show that something is loading.

**Review by Simplypheyie96 — APPROVED** [review]

**Comment by Adebesin-Cell** [comment]
> @Adebesin-Cell , loading this page https://oraqles-908uo3d0r-prediqt.vercel.app/oracles/march-madness for the first time takes a lot of time, maybe you should add a loader to show that something is loading.

Screen Shot 2022-12-29 at 12 35 09 PM

There's a loading state going on to preload the data, I think it's still related with the large data rendering.

**Pull request merged**
